geometry_msgs/Point first_point # First point of line segment
geometry_msgs/Point last_point # Last point of line segment
